sphinx-ecosystem-news-january-2022.rst

```diff
@@ -0,0 +1,115 @@
+   enables documentation writers to control
+   when intersphinx references should be used.
+
+   You can use the new ``:external:`` role as follows:
+
+   .. code-block:: rst
+
+      External reference: :external:py:class:`zipfile.ZipFile`.
+      External reference with constrained domain lookup: :external+python:py:class:`zipfile.ZipFile`.
+
+   See more information in :rst:role:`the official documentation <external>`.
+
+New asynchronous methods to load JavaScript files
+   The method :py:meth:`sphinx:sphinx.application.Sphinx.add_js_file`
+   now accepts a new ``loading_method`` parameter that can either be ``"async"`` or ``"defer"``,
+   which results in the script being loaded with the
+   `async <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async>`_ or
+   `defer <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer>`_
+   HTML attributes respectively.
+
+Proper error messages when autosummary does not find a package
+   Sphinx ships the :std:doc:`autosummary <sphinx:usage/extensions/autosummary>` extension
+   to automatically generate API pages of a Python library.
+   However, autosummary used to raise misleading error messages if the target library failed to import,
+   which resulted in user frustration.
+   This problem has now been fixed and autosummary properly points out the reason of the ``ImportError``.
+
+You can read the `complete Sphinx 4.4.0 release
+notes <https://www.sphinx-doc.org/en/master/changes.html#release-4-4-0-released-jan-17-2022>`_ online.
+
+**pydata-sphinx-theme**, a Sphinx theme based on Bootstrap CSS developed by the PyData community,
+`got a new 0.8 release <https://github.com/pydata/pydata-sphinx-theme/releases/tag/v0.8.0>`_
+on January 15th. The Read the Docs team collaborated with the developers
+to make some of the new features behave correctly in our platform, in particular
+`the custom version
+switcher <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#add-a-dropdown-to-switch-between-docs-versions>`_.
+As `Chris Holdgraf summarizes in this Twitter
+thread <https://twitter.com/choldgraf/status/1482435411301449729>`_,
+the new version brings other interesting additions,
+like `better depth control of the left
+sidebar <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#navigation-depth-and-collapsing-of-the-sidebar>`_
+and `custom SVG
+icons <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#local-image-icons>`_.
+
+Extensions
+----------
+
+Several new extensions saw new releases recently, including:
+
+`sphinx-codeautolink 0.9 <https://sphinx-codeautolink.readthedocs.io/en/stable/release_notes.html#id2>`_
+   sphinx-codeautolink is a Sphinx extension to automatically include links inside code blocks.
+   The 0.9 version now links Python builtin objects if intersphinx is properly configured
+   and has several logging improvements.
```

Comment:
We should be promoting this extension more! It seems super useful.

Reply:
Yep: https://github.com/humitos/sphinx-extensions/issues/18

Reply:
cool - I hadn't heard about this before either!